go/l1client/txhandler/data_handler.go

```diff
@@ -0,0 +1,48 @@
+package txhandler
```

Comment:
maybe rename this file to "utils.go"

Reply:
Felt the same, it's not a great name, I had renamed to `input_handler` as it handles the smartcontract inputs. What do you think ?

I would love to have some name that's descriptive if that's ok.